### PR TITLE
App 739, Configure New Component

### PR DESCRIPTION
**Pull request by pdarde-daitan**
Configure New Component
https://perzoinc.atlassian.net/browse/APP-739

### CODE REVIEW

File: integration-fe/src/js/app.jsx [hunk 1]
```diff
@@ -34,7 +35,10 @@ let themeSize;
 function loadApplication(rooms) {
   render(
     <Provider store={store}>
-      <Home />
+      <Router history={hashHistory}>
```
Comment (cmarcondes-daitan) [inline integration-fe/src/js/app.jsx]: isn't a good idea to create an Router component, to make it split ?

File: integration-fe/src/views/CreateView.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class CreateView extends Component {
+
+  componentDidMount() {
```
Comment (cmarcondes-daitan) [inline integration-fe/src/views/CreateView.jsx]: It could be removed.

**Review by cmarcondes-daitan — CHANGES_REQUESTED** [review]

File: integration-fe/src/js/app.jsx (hunk 1 above)
Comment (pdarde-daitan) [reply integration-fe/src/js/app.jsx]: Good Idea. I will do.

File: integration-fe/src/views/CreateView.jsx (hunk 2 above)
Comment (pdarde-daitan) [reply integration-fe/src/views/CreateView.jsx]: For sure. I've created this method to avoid warnings from es lint. I will change this.

**Review by cmarcondes-daitan — APPROVED** [review]

**Pull request merged**